tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -137,6 +138,11 @@ public void processNe(SourceFeature sf, FeatureCollector features) {
         // since all these are locality, we hard code kindRank to 2 (needs to match OSM section below)
         .setSortKey(getSortKey(minZoom, 2, populationRank, population, sf.getString("name")));
 
+      var script = Script.getScript(sf.getTag("name").toString());
+      if (!script.equals("LATIN") && !script.equals("GENERIC")) {
+        feat.setAttr("pmap:script", script);
```

Comment:
Generally we prefer lowercase keys and values elsewhere in the tile schema.

But if the [ISO standard](https://unicode.org/iso15924/iso15924-codes.html) prefers Titlecases then that's OK. But I don't see UPPERCASE in the standard.

Reply:
This Java library returns UPPERCASE constants for the various scripts. So I lean towards lowercasing them in our export. Something like:

> var script = Script.getScript(sf.getTag("name").toString().toLowerCase());

Reply:
Good point. I checked on Wikipedia and there they use always first letter uppercase followed by lowercase letters, e.g., "Devanagari". This makes sense to me since languages are similarly capitalized. I adopted the code.